### PR TITLE
UAF-5298 - Merge v6 Release 31 Into our current v7 git repos, both KFS and Rice

### PR DESCRIPTION
**Pull request by sskinner**
This PR merges rice-2.1-ua-master into rice-2.5-ua-development, which forward-ports the KFS6 release 31 into our KFS7 rice repo. Please see the related jira for a full breakdown.

### CODE REVIEW

File: db/impex/master/pom.xml [hunk 1]
```diff
@@ -40,7 +40,6 @@
     <profile>
       <id>master</id>
       <properties>
-        <impex.username>RICE_${project.version.sanitized}</impex.username>
```
Comment (nataliac06) [inline db/impex/master/pom.xml]: I don't see any reason why we would need to remove this property... I'd leave it in.

File: db/pom.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright 2005-2016 The Kuali Foundation
+    Copyright 2005-2015 The Kuali Foundation
```
Comment (nataliac06) [inline db/pom.xml]: Nit pick: no reason to change this file

File: db/impex/master/pom.xml (hunk 1 above)
Comment (sskinner) [reply db/impex/master/pom.xml]: Good catch, I'll add this back in.

File: db/pom.xml (hunk 2 above)
Comment (sskinner) [reply db/pom.xml]: Auto-merge, I'll revert.

File: db/impex/master/pom.xml (hunk 1 above)
Comment (sskinner) [reply db/impex/master/pom.xml]: Added back in.

**Comment by nataliac06** [comment]
Looks ok.

**Pull request merged**